boards/nxp/frdm_rw612/doc/index.rst

```diff
@@ -36,8 +36,6 @@ Supported Features
 +-----------+------------+-----------------------------------+
 | USART     | on-chip    | serial                            |
 +-----------+------------+-----------------------------------+
-| BLE       | on-chip    | Bluetooth                         |
```

Comment:
why is this deleted ?

Reply:
it’s duplicated, you can expand the hidden lines

Reply:
ok, you can mention this in commit message